test/testConditionalSections.js

```diff
@@ -441,7 +453,7 @@ Or that one at the same time, because they are both part of an A/B test.
     // format, section metadata, the mdast format or the input above, you probably
     // want to change one of these values until you find one that *happens* to work
     // again...
-    assert.notDeepEqual(await runpipe('foo'), await runpipe('bang'));
+    assert.notDeepEqual(await runpipe('foo'), await runpipe('bar'));
```

Comment:
As per existing comment above, the test started failing after my changes, so tried out some other value